CHANGES.rst

```diff
@@ -10,6 +10,10 @@ Announcements
 ^^^^^^^^^^^^^
 * `xclim` has migrated its development branch name from `master` to `main`. (:issue:`1667`, :pull:`1669`).
 
+Breaking changes
+^^^^^^^^^^^^^^^^
+* Added the `tox-gh` dependency to the development installation recipe. This will soon be required for running the `tox` test ensemble on GitHub Workflows. (:pull:`1709`).
```

Comment:
Honestly, I don't think this counts as a "breaking change".  More of an "internal change" no ?
I understand "breaking" as targeting a normal usage, not a development setup.

Reply:
Good point. We need to have a defining document on what exactly is a "breaking change".